ros2service/ros2service/verb/call.py

```diff
@@ -16,15 +16,26 @@
 import time
 
 import rclpy
+from rclpy.qos import QoSDurabilityPolicy
+from rclpy.qos import QoSProfile
+from rclpy.qos import QoSReliabilityPolicy
 from ros2cli.node import NODE_NAME_PREFIX
 from ros2service.api import ServiceNameCompleter
 from ros2service.api import ServicePrototypeCompleter
 from ros2service.api import ServiceTypeCompleter
 from ros2service.verb import VerbExtension
+from ros2service.api import profile_configure_short_keys
 from rosidl_runtime_py import set_message_fields
 import yaml
 
 
+def get_pub_qos_profile():
+    return QoSProfile(
+        reliability=QoSReliabilityPolicy.RELIABLE,
+        durability=QoSDurabilityPolicy.TRANSIENT_LOCAL,
+        depth=1)
+
+
```

Comment:
Why we need to statically declare default profile only for `ros2 service call`? instead, can we use `rclpy.qos.QoSPresetProfiles.get_from_short_key(qos_profile_name)` and qos_profile_name is set `default` in default?

Reply:
I will have a look into this. Thanks